src/frontend/src/lib/components/ui/LogoWithCorner.svelte

```diff
@@ -0,0 +1,24 @@
+<script lang="ts">
+	import Logo from '$lib/components/ui/Logo.svelte';
+	import type { Token } from '$lib/types/token';
+
+	export let token: Token;
+	export let size: string;
+	export let color: 'dust' | 'off-white' | 'white' = 'dust';
+	export let showNetworkIcon = true;
+
+	const {
+		icon,
+		name,
+		network: { name: networkName, icon: networkIcon }
+	} = token;
+</script>
+
+<div class="relative">
+	<Logo src={icon} alt={`${name} logo`} {size} {color} />
```

Comment:
Unrelated to this PR: should we i18n `... logo`?

Reply:
yes! good point! i will do in another PR

Reply:
Created PR #1431 for this